db/schema.rb

```diff
@@ -65,6 +65,13 @@
     t.index ["receiver_id", "receiver_type"], name: "index_mailboxer_receipts_on_receiver_id_and_receiver_type"
   end
 
+  create_table "tests", force: :cascade do |t|
+    t.string "email"
+    t.string "password"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
Why is this created?

Reply:
Incorrectly created a second database, dropped it now and migrated db again